docs/configuration/modules/remote-service.md

```diff
@@ -3,9 +3,9 @@ title: remote-service
 layout: default
 ---
 
-The `remote-service` module is used by [AFS]() to define a remote AFS server.
+The `remote-service` module is used by [AFS](../../todo.md) to define a remote AFS server.
```

Comment:
this page for AFS exists, redirect to it instead:
`../../afs/index.md`

Reply:
Change to `../../afs/afs-remote.md` seems better. What do you think?